### PR TITLE
cmd/action: document SQLiteDB and MariaDB

### PR DESCRIPTION
**Pull request by yonidavidson**

### CODE REVIEW

File: cmd/action/apply.go [hunk 1]
```diff
@@ -25,7 +25,9 @@ var (
 		Run:   CmdApplyRun,
 		Example: `
 atlas schema apply -d mysql://user:pass@tcp(localhost:3306)/dbname -f atlas.hcl
-atlas schema apply --dsn postgres://user:pass@host:port/dbname -f atlas.hcl`,
+atlas schema apply -d maria://user:pass@tcp(localhost:3306)/dbname -f atlas.hcl
+atlas schema apply --dsn postgres://user:pass@host:port/dbname -f atlas.hcl
+atlas schema apply -d sqlite3://file:ex1.db?_fk=1 -f atlas.hcl`,
```
Comment (zeevmoney) [inline cmd/action/apply.go:30]: are we ok with the fact that this is not a standard sqlite connection string?

Comment (zeevmoney) [reply cmd/action/apply.go:30]: https://github.com/mattn/go-sqlite3#dsn-examples

File: README.md [hunk 2]
```diff
@@ -6,7 +6,9 @@ be used individually but are designed to work very well together.
 
 ### Supported databases: 
 * MySQL
+* MariaDB
 * PostgresSQL
+* SQLiteDB
```
Comment (a8m) [inline README.md:11]:
```suggestion
* SQLite
```

File: cmd/action/apply.go (hunk 1 above)
Comment (a8m) [inline cmd/action/apply.go:30]:
```suggestion
atlas schema apply -d sqlite://file:ex1.db?_fk=1 -f atlas.hcl`,
```

**Review by a8m — APPROVED** [review]
👌🏻

**Pull request merged**